tools/scoring/score.py

```diff
@@ -102,6 +106,7 @@ def log(string):
 
 # Split a benchmark string into benchmark, division and family.
 def split_benchmark_division_family(x, family_func):
+    assert len(x) == 2, "weird x: {0}".format(x)
```

Comment:
I switched to format strings for new python scripts:  `f"weird x: {x}"` (without .format(x)).  The error message could also be improved :)